.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "\U0001F680 Feature Request"
+about: I have a suggestion
+
+---
+
+## Feature Request
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
The next question sort of assumes the answer to this is "Yes."

If the user says "No, it's not." and goes to the next question it is asking about a solution to this question, which they answered no to. Perhaps making the next question "**Describe the feature you'd like**" would improve that flow.